components/brave_wallet_ui/components/desktop/lock-screen/index.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react'
+
+import {
+  StyledWrapper,
+  Title,
+  IconBackground,
+  PageIcon,
+  InputColumn,
+  Input
+} from '../wallet-onboarding/create-password/style'
+import { NavButton } from '../../extension'
+import locale from '../../../constants/locale'
+
+export interface Props {
+  action: () => void
+  password: (event: any) => void
+  disabled: boolean
+}
+
+const LockScreen = (props: Props) => {
+  const { action, password, disabled } = props
+
+  return (
+    <StyledWrapper>
+      <IconBackground>
+        <PageIcon />
+      </IconBackground>
+      <Title>{locale.lockScreenTitle}</Title>
+      <InputColumn>
+        <Input type='password' placeholder={locale.createPasswordInput} onChange={password} />
```

Comment:
oof, lol good catch!! I think it would be best if I make this change in a new PR and knock it all out in one push for all of the wallet components will need to be updated to this.